src/CMakeLists.txt

```diff
@@ -473,8 +473,13 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
 endif()
 
 if (USE_OPENGLES)
-    find_library(OPENGLES_LIBRARY GLESv3)
-    target_link_libraries(libultraship PRIVATE GLESv3)
+    if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+        find_library(OPENGLES_LIBRARY GLESv2)
+        target_link_libraries(libultraship PRIVATE GLESv2)
+    else()
+        find_library(OPENGLES_LIBRARY GLESv3)
+        target_link_libraries(libultraship PRIVATE GLESv3)
```

Comment:
in the PR description you mention

> Also, it deals with the fact that -lGLESv3 does not exist on GNU/Linux, where GLES3 programs are built with -lGLESv2 because GLES3 is considered simply a GLES2 extension.

if that is the case, then why is GLESv3 still being used outside of linux here?

Reply:
ah, GLESv3 is an android (only?) thing, got it